### DISCUSSION TITLE
I was running train and model_devi with one cluster and fp with another cluster and had the following problem

### DISCUSSION THREAD

**Discussion by sooyaaa233** [deepmodeling/dpgen#1206] [Q&A]
![image](https://user-images.githubusercontent.com/126163538/236385924-9523c2fb-8d3b-4aba-80a1-673482d86939.png)
![image](https://user-images.githubusercontent.com/126163538/236386007-1bb423a6-6ec4-4411-86a4-9b9b16c45215.png)
![image](https://user-images.githubusercontent.com/126163538/236386021-29c5b327-24e9-4260-9f44-f82d5c3698e0.png)

**Comment by sooyaaa233** [comment]
sorry，This is the problem with dpgen

**Comment by njzjz** [comment]
For Fig. 2, it seems that `qsub` command is not found?

**Reply by sooyaaa233** [reply]
It looks like this, but sub is working for me in the second cluster, I asked my classmate and the second cluster is indeed using PBS. but, after I changed the PBS in the fp section of machine.json to Shell yesterday, it works, I don't understand why
